src/phase/utils/phase_io.py

```diff
@@ -49,6 +49,10 @@ def __init__(self, init=True, pss=None, host=None):
         app_secret = pss
         self._api_host = host
 
+        # Check if app_secret is None
+        if app_secret is None:
+            raise ValueError("Phase token (pss) cannot be None")
```

Comment:
```suggestion
            raise ValueError("Phase token (pss) is missing")
```

This seems like a more useful exception message

Reply:
Wanted to stick with python lingo as the SDK is facing a technical user.

Reply:
Sure, but a user of the SDK is likely to encounter this exception when they either dont set the right env var, or don't supply the token in their code. "Cannot be None" doesn't seem to indicate that, it's not not clearly actionable IMO

Reply:
Feel free to resolve if you think it's fine as is